src/rebalance.go

```diff
@@ -18,7 +18,7 @@ func rebalance(a *App, req RebalanceRequest) bool {
   // find the volumes that are real
   rvolumes := make([]string, 0)
   for _, rv := range req.volumes {
-    if remote_head(fmt.Sprintf("http://%s%s", rv, kp)) {
+    if remote_head(fmt.Sprintf("http://%s%s%s", a.userpass, rv, kp)) {
```

Comment:
This change is now on four lines, it should probably be abstracted

Reply:
I tried it by passing userpass into the volumes name, which would mean no change would have to be done to rebalance and rebuild. This doesn't work because userpass ends up being written to levelDB which is unwanted behaviour. (leveldb_compare.go fails)

Reply:
Hmm, I see. Okay, this seems fine.